Review & finalize 8.3

AzharAlali opened the pull request:
Review & finalize 8.3

denisecase committed:
Merge pull request #68 from denisecase/revert-67-patch-1

Revert "written answer to the question 8.6"

harsha4824 committed:
answered issue no 8.4

denisecase committed:
Merge pull request #69 from harsha4824/patch-1

answered issue no 8.4

rohithbharadwaj committed:
edited 8.5

santhoshkumarbollena committed:
Merge pull request #71 from rohithbharadwaj/patch-2

edited 8.5

rohithbharadwaj committed:
Update 1-8.md

santhoshkumarbollena committed:
Merge pull request #72 from rohithbharadwaj/patch-3

Update 1-8.md

SamFields committed:
Completed Issue 8.7

santhoshkumarbollena committed:
Merge pull request #73 from SamFields/patch-1

Completed Issue 8.7

mist543 committed:
Updated 8.8 task

rohan6471 committed:
Merge pull request #74 from Chaitra543/patch-1

Updated 8.8 task

reddylavanya committed:
updated on 8.9

Samanthgourineni committed:
Update to 8.11

Made changes to the draft

rohan6471 committed:
Merge pull request #78 from Samanthgourineni/patch-2

Update to 8.11

Rahulmallampati825 committed:
Answered the 6.3 question

Prajakt-Khawase committed:
How to make a new custom producer,

Completed issue 8.12

sneharao0806 committed:
final update on 8.10

santhoshkumarbollena committed:
Merge pull request #85 from sneharao0806/patch-2

final update on 8.10

santhoshkumarbollena committed:
Merge pull request #76 from reddylavanya/patch-1

updated on 8.9

nandiniyadavjanga committed:
Update 6.4

sunilmundru committed:
Answered 6.10 by sunil

sunilmundru committed:
updated 6.10

sunilmundru committed:
updated 6.10

SOWMYAREDDY97 committed:
8.6 edited

rohan6471 committed:
Merge pull request #87 from nandiniyadavjanga/patch-1

Update 6.4

rohan6471 committed:
Merge pull request #84 from Prajakt-Khawase/patch-2

How to make a new custom producer,

rohan6471 committed:
Merge pull request #91 from SOWMYAREDDY97/patch-1

8.6 edited

rohan6471 committed:
Merge pull request #88 from sunilmundru/master

Answered question 6.10

DeepakJayanthi committed:
Updated the 8.4 question and verified.

reddylavanya committed:
8.9 final commit

santhoshkumarbollena committed:
Merge pull request #94 from reddylavanya/patch-3

8.9 final commit

Leelakrishna0981 committed:
update in 1.8

HarshithGudapati committed:
Reviewed the answer

bhuvaneshnakka committed:
udated 8.7

Nishanthreddy1997 committed:
modified answer 6.11. and finalized with adding some more content.

ganesh58 committed:
Update 1-7.md

BgopikrishnaSabbineni committed:
Update 1-6.md

santhoshkumarbollena committed:
Merge pull request #102 from BgopikrishnaSabbineni/patch-2

Update 1-6.md

denisecase committed:
Merge pull request #93 from DeepakJayanthi/patch-1

Updated the 8.4 question and verified.

denisecase committed:
Merge pull request #96 from Leelakrishna0981/patch-2

update in 8.10 answer

denisecase committed:
Merge pull request #97 from HarshithGudapati/patch-1

Reviewed the answer 7.1

denisecase committed:
Merge pull request #98 from bhuvaneshnakka/patch-1

8.7

denisecase commented:
Is this on stale code?  How about starting over - it appears you may have made edits on very old code?

denisecase committed:
Merge pull request #100 from Nishanthreddy1997/patch-1

modified answer 6.11. and finalized with adding some more content.

denisecase committed:
Merge pull request #101 from ganesh58/patch-1

Update 1-7.md

denisecase commented:
Please resolve your conflicts so you don't lose anyone else's work. I suggest you delete and perform your changes on fresh code, quickly, as described in the instructions.

milindar committed:
updated 6.3 issue

denisecase committed:
Merge pull request #103 from milindar/patch-2

updated 6.3 issue

Nishanthreddy1997 committed:
updated 6.11.

karthikmuthyala committed:
Review the answer 1.6

denisecase committed:
Revert "8.7"

denisecase committed:
Merge pull request #106 from denisecase/revert-98-patch-1

Revert "8.7"

denisecase committed:
Merge pull request #105 from karthikmuthyala/patch-1

Review the answer 1.6

denisecase committed:
Merge pull request #104 from Nishanthreddy1997/patch-2

updated 6.11.

bharadwaj1995 committed:
adding review for question 8.8

added logic description for code

denisecase committed:
Merge pull request #107 from bharadwaj1995/patch-2

adding review for question 8.8

Rahulmallampati825 committed:
Merge branch 'master' into master

nikitha0012 committed:
8.9 - wrote review for the answer

8.8's review answer is bit similar to my review. Wrote detailed review for 8.9.

Rahulmallampati825 committed:
Reviewed and Commited the above 6.3 question

Telugua committed:
I reviewed the issue 6.8 and finalized

Akhilreddy17 committed:
Reviewed the 6.7 issue and finalized.

StephenBurke committed:
added 6.6 answer

denisecase committed:
Merge pull request #108 from NikithaLakmarapu/patch-2

8.9 - wrote review for the answer

denisecase committed:
Merge pull request #112 from StephenBurke/master

added 6.6 answer

denisecase committed:
Merge pull request #110 from Telugua/patch-1

I reviewed the issue 6.8 and finalized

denisecase committed:
Merge pull request #111 from Akhilreddy17/patch-1

Reviewed the 6.7 issue and finalized.

davulurirevanth committed:
I reviewed 8.11 and finalized the issue

bhuvaneshnakka committed:
adding review and code explanation for question 8.10

denisecase committed:
Merge pull request #114 from davulurirevanth/patch-1

I reviewed 8.11 and finalized the issue

denisecase committed:
Merge pull request #115 from bhuvaneshnakka/patch-1

adding review and code explanation for question 8.10

denisecase committed:
Merge pull request #82 from Rahulmallampati825/master

Answered the 6.3 question
